depro-inactive-guests.js

```diff
@@ -0,0 +1,205 @@
+            console.log("No one will be deprovisioned.");
+        }
+    });
+};
+
+const main = () => {
+    // 1. Take options if we have them
+    commander
+        .version("0.0.3")
+        .option("-a, --auth <token>", "Service account token")
+        .option("-c, --csv <path>", "Path to member csv file")
+        .option("-o, --organization_id <organization_id>", "Organization or Workspace domain ID")
+        .option("-t, --threshold <#>", "# of days inactive threshold", parseInt)
+        .option("-m, --mode ['dry' or 'action']", "mode to run the script in")
+        .parse(process.argv);
+
+    if (commander.rawArgs.length > 2) {
+        // We've been given options, lets check those and then process those through
+        if (!commander.auth) {
+            console.log("Please set the `--auth` option with a service account token");
+            return;
+        }
+        if (!commander.csv) {
+            console.log("Please set the`--csv` option with a path or URL to a CSV file");
+            return;
+        }
+        if (!commander.organization_id) {
+            console.log("Please set a the `--domain` option with your domain ID");
+            return;
+        }
+        if (!commander.threshold) {
+            console.log(
+                "Please set a the `--threshold` option with the number of days of inactivity you'd like to count as being inactive"
+            );
+            return;
+        }
+        if (commander.mode !== "action") {
+            console.log(
+                "In dry-run mode. set --mode option to 'action' to actually deprovision users"
+            );
+        }
+
+        processInputs(commander);
+    } else {
+        // If we have no options, prompt the user for the inputs we need.
+        const schema = {
+            properties: {
+                auth: {
+                    message: "please enter a valid service account token",
+                    pattern: /^0\//,
+                    required: true
+                },
+                csv: {
+                    message: "absolute path or url to csv file",
+                    required: true
+                },
+                organization_id: {
+                    message: "please enter a valid organization id",
+                    pattern: /^[1-9]\d*$/,
+                    required: true
+                },
+                threshold: {
+                    message: "number of days inactive",
+                    required: true
+                },
+                mode: {
+                    message: "mode (dry or action)",
+                    required: false
+                }
+            }
+        };
+
+        // 2. Prompt user for inputs
+        prompt.colors = false;
+        prompt.message = "";
+        prompt.start();
+
+        prompt.get(schema, (err, inputs) => {
+            if (err) {
+                throw err;
+            }
+            processInputs(inputs);
+        });
+    }
+};
+
+main();
```

Comment:
I tried running npm test, and everything passed, but afterwards prompted me for a service account token. I think it's because main() is run just by loading this 

I'm not sure what the traditional way to avoid that is? Maybe a separate file that tests load, which isn't an entry point?